ldapauthenticator/ldapauthenticator.py

```diff
@@ -28,7 +28,7 @@ def _server_port_default(self):
         help='Use SSL to encrypt connection to LDAP server'
     )
 
-    bind_dn_template = Unicode(
+    bind_dn_template = List(
```

Comment:
I think we should allow this to continue to be a Unicode, to prevent backwards compatibility issues. This could be done with a [Union](http://traitlets.readthedocs.io/en/stable/trait_types.html#traitlets.Union) traitlet type I think.